wasm-build.sh

```diff
@@ -16,7 +16,6 @@ case $BUILD_MODE in
         ;;
     "dev")
         BUILD_FLAG="--dev"
-        BUILD_MODE_ARGS="${BUILD_MODE_ARGS},console_log"
```

Comment:
Since we no longer include the console_log just for dev build, we don't need to re-bind the BUILD_MODE_ARGS